iphone/Maps/Classes/CarPlay/CarPlayService.swift

```diff
@@ -40,19 +40,6 @@ final class CarPlayService: NSObject {
     self.interfaceController?.delegate = self
     let configuration = CPSessionConfiguration(delegate: self)
     sessionConfiguration = configuration
-    // Try to use the CarPlay unit's interface style.
-    if #available(iOS 13.0, *) {
-      switch configuration.contentStyle {
-      case .light:
-        rootTemplateStyle = .light
-        window.overrideUserInterfaceStyle = .light
-      case .dark:
-        rootTemplateStyle = .dark
-        window.overrideUserInterfaceStyle = .dark
-      default:
-        rootTemplateStyle = window.overrideUserInterfaceStyle == .light ? .light : .dark
```

Comment:
Was this case ever used? Is it handled now?

Reply:
We should not use `window.userInterfaceStyle` - this property is responsible for the car play UI,  not `content`. Important to separate this terms. 
We should only response to the contentTheme (templates with arrows, estimates etc). 
Content theme have only 2 cases: dark and light.

![image](https://github.com/organicmaps/organicmaps/assets/79797627/4a0f74f9-fab3-42d6-bfb4-9267681a58c9)

Reply:
It used to work fine in CarPlay before. Hope this change won't break it.

Reply:
I followed the documentation and tested a lot. Think it should work well!

> The vehicle selects the content style according to the ambient light level. Your navigation app can use this value to determine the most appropriate style of **map content to draw** in its base view. The **content style is independent of the user interface style**, which controls light and dark mode.

https://developer.apple.com/documentation/carplay/cpsessionconfiguration/contentstyle?language=objc